docs/1 Introduction/3 Quickstart.md

```diff
@@ -1,21 +1,45 @@
 # Quickstart
 
-With `starport` installed on your machine, you can now build your very first blockchain!
+Use these three commands to build, start, and define your first blockchain.
+
+## Create a blockchain
+
+To create a blockchain:
 
 ```
 starport app github.com/username/myapp && cd myapp
 ```
 
-This command will create a directory `myapp` and scaffold a Cosmos SDK blockchain.
+The `app` command creates the blockchain directory `myapp` and scaffolds a [Cosmos SDK](https://docs.cosmos.network/) blockchain.
+
+## Run a blockchain
+
+To run a blockchain in your development environment:
 
 ```
 starport serve
 ```
 
-`serve` will install dependencies, build, initialise and run your blokchain.
+The `serve` command installs dependencies, builds, initializes, and starts the blockchain.
+
+## Define
+
+To add a custom type with create, read, update, and delete (CRUD) functionality:
 
 ```
 starport type post title body
 ```
 
-`type` scaffolds functionality to create, read, update and delete for a custom type.
\ No newline at end of file
+The `type` command scaffolds functionality a custom type.
+
```

Comment:
## Next steps

To learn more about building Cosmos SDK blockchains with Starport, try a beginner-friendly [IBC Hello world tutorial](http://tutorials.cosmos.network/hello-world/tutorial/).